padlockcloud/sender.go

```diff
@@ -27,21 +43,89 @@ type EmailSender struct {
 
 // Attempts to send an email to a given recipient. Through `smpt.SendMail`
 func (sender *EmailSender) Send(rec string, subject string, body string) error {
-	auth := smtp.PlainAuth(
-		"",
-		sender.Config.User,
-		sender.Config.Password,
-		sender.Config.Server,
-	)
-
-	message := fmt.Sprintf("Subject: %s\r\nFrom: Padlock Cloud <%s>\r\n\r\n%s", subject, sender.Config.User, body)
-	return smtp.SendMail(
-		sender.Config.Server+":"+sender.Config.Port,
-		auth,
-		sender.Config.User,
-		[]string{rec},
-		[]byte(message),
-	)
+
+	// Send Email using SMTP Protocol
+	if strings.ToLower(sender.Config.Provider) == "smtp" {
+		//fmt.Println("Sending Email Using SMTP.")
+		log.Print("Using SMTP For Sending Email.")
```

Comment:
I'm assuming you added this for debugging purposes? If you actually do want to log something here please make add more information/context and make sure to use the same pattern as the rest of the app, i.e use the `Log` struct and similar formatting. Example: https://github.com/MaKleSoft/padlock-cloud/blob/master/padlockcloud/handler.go#L149